ui/js/treeherder.js

```diff
@@ -2,4 +2,5 @@
 
 /*exported treeherder*/
 module.exports = angular.module('treeherder',
-    ['ngResource', 'ngSanitize', 'ngCookies', 'LocalStorageModule']);
+    ['ngResource', 'ngSanitize', 'ngCookies', 'LocalStorageModule'])
+    .constant('pinboardError', 'Max pinboard size of 500 reached.');
```

Comment:
We don't religiously observe the convention in Treeherder, but I think we still strive to.. external customer facing strings in the app are double-quoted in js, and internal strings are single-quoted. I defer to @camd if that's changed :)

For the case above that would be:

`'pinboardError', "Max pinboard size of 500 reached."`

Reply:
I don't have a strong preference here, tbh.  But what @tojon says seems reasonable.  :)

Reply:
This is a totally reasonable way to add this constant.  But we have generally added these in a file called ``values.js``.  I realize this is a constant as opposed to a value.  Though most (all?) of those values in ``values.js`` should likely be constants.  Let's add this there, if you would.  You can still make it a constant, though.  We could open a bug to convert those values to constants at some point.  Though as we transition to React from Angular, it will become moot.